src/portable/ver.rs

```diff
@@ -212,6 +213,16 @@ impl Specific {
     pub fn is_stable(&self) -> bool {
         matches!(self.minor, MinorVersion::Minor(_))
     }
+
+    pub fn slot(&self) -> String {
+        match self.minor {
+            MinorVersion::Minor(_) => self.major.to_string(),
+            MinorVersion::Dev(v) => format!("{}-dev{}", self.major, v),
+            MinorVersion::Alpha(v) => format!("{}-alpha{}", self.major, v),
+            MinorVersion::Beta(v) => format!("{}-beta{}", self.major, v),
+            MinorVersion::Rc(v) => format!("{}-rc{}", self.major, v),
+        }
+    }
```

Comment:
I'm not sure if I understand what's a "slot".

Reply:
That's how extensions are linked to server builds -- see the tags in the JSON manifest. I have a feeling we'll want to use semver ranges instead.

https://packages.edgedb.com/archive/.jsonindexes/x86_64-unknown-linux-gnu.json